docs/Language/C++/C++.md

```diff
@@ -9,9 +9,135 @@ has_children: true
 {{ page.title }}
 ======================
 
-{% include under_construction.html %}
+C++ is a general-purpose programming language that was developed as an extension of the C programming language. It was created by Bjarne Stroustrup at Bell Labs in the early 1980s. C++ introduces object-oriented programming features to C, allowing developers to write more organized and modular code.
 
+-----------------------------------------------------------------------
 
-<br>
+### Official Documentation
 
-<br>
\ No newline at end of file
+- Microsoft Documentation: https://learn.microsoft.com/en-us/cpp/cpp/
```

Comment:
Should be made into a clickable link Consider adding [https://learn.microsoft.com/en-us/cpp/cpp/](https://learn.microsoft.com/en-us/cpp/cpp/) Otherwise it remains as a text 'without linking.

Reply:
Fixed